lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -31,7 +31,9 @@ class LoadFastEnough4Pwa extends Audit {
       category: 'PWA',
       name: 'load-fast-enough-for-pwa',
       description: 'Page load is fast enough on 3G',
-      helpText: 'Satisfied if First Interactive is less than 10 seconds, as defined by the [PWA Baseline Checklist](https://developers.google.com/web/progressive-web-apps/checklist). Network throttling is required (specifically: RTT latencies >= 150 RTT are expected).',
+      helpText: 'Ensuring that page load is fast enough on a 3G network ensures that ' +
```

Comment:
that works for me. 

kayce can you add something like ` Network throttling is required (specifically: RTT latencies >= 150 RTT are expected)` to the doc page?  This is an important detail. :)

side note: we'll have to make sure the tone of these are consistent.. e.g. if we always go with a matter-of-fact sentence rather than describing the audit.